### ISSUE TITLE
False positive for variable set but is never used

### ISSUE THREAD

**Issue by iBicha** [rokucommunity/bslint#100]
```
    j = 0
    while true
        j += 1
        if j > 10
            j = 0 ' lint error here
        end if
    end while
```

```
[{
	"resource": "file.bs",
	"owner": "_generated_diagnostic_collection_name_#0",
	"code": "LINT1005",
	"severity": 8,
	"message": "Variable 'j' is set but value is never used",
	"source": "brs",
	"startLineNumber": 85,
	"startColumn": 13,
	"endLineNumber": 85,
	"endColumn": 14
}]
```
